types/block.model.ts

```diff
@@ -0,0 +1,26 @@
+import { Field } from './event.model';
+import { Regatta } from './regatta.model';
+import { RoundDetail } from './round.model';
+
+export interface Block {
+	id: string; // Zijn alle '_oud' ids nog nodig?
+	regatta_id: string;
+	block: number;
+	start_time: Date;
+	status: number; // Wat zijn de waarden van deze statussen?
```

Comment:
Zie: https://github.com/Proteus-Eretes/IRIS2/blob/develop/src/Model/Types/Block/BlockStatus.php

Als je het niet kan zien:

```
 public const CREATED = 1;
    public const ROUNDS_ASSIGNED = 2;
    public const EVENTS_ASSIGNED = 4;
    public const START_ORDER = 8;
    public const TOSSED = 16;
    public const SHIRT_NUMBERS_ASSIGNED = 32;

    public const BLOCK_STATUS = [
        self::CREATED => 'Created',
        self::ROUNDS_ASSIGNED => 'Rounds assigned',
        self::EVENTS_ASSIGNED => 'Events assigned',
        self::START_ORDER => 'Starting order determined',
        self::TOSSED => 'Lots drawn',
        self::SHIRT_NUMBERS_ASSIGNED => 'Shirt numbers assigned',
    ];
```

Reply:
Ah ja, bdeankt! Ik zal sws de types in die map even overnemen